src/main/java/com/example/marspictmp/controller/NasaFeignClient.java

```diff
@@ -0,0 +1,16 @@
+package com.example.marspictmp.controller;
+
+import org.springframework.cloud.openfeign.FeignClient;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+
+import com.fasterxml.jackson.databind.JsonNode;
+
+@FeignClient(name = "nasa", url = "${nasa.url}")
```

Comment:
I think It is better to set the "name" property as a value from application properties to have a feign configuration in one place.